modules/meeting/app/components/meetings/update_flash_component.html.erb

```diff
@@ -1,21 +1,40 @@
 <%=
   component_wrapper do
-    flex_layout do |flex|
-      flex.with_row(classes: "op-toast--wrapper") do
-        render(Primer::Alpha::Banner.new(
-          full: false,
-          icon: 'info',
-          scheme: :default,
-          data: {
-            singleton: true
-          }
-        )) do |component|
-          component.with_action_button(
-              tag: :a,
-              href: meeting_path(@meeting),
-              data: { turbo: false },
-              size: :medium) { I18n.t("label_meeting_reload") }
-          I18n.t("notice_meeting_updated")
+    if @button
+      flex_layout do |flex|
```

Comment:
@oliverguenther do you know if this is the correct way to style this? I think we may have missed something before because when I trigger a full width flash, it stays the same width but in the full style and doesn't expand like I was assuming it would

Not sure if using flex with the `op-toast--wrapper` is working as intended tbh

Reply:
@HDinger same question :)